pyogrio/tests/test_raw_io.py

```diff
@@ -363,3 +363,47 @@ def test_read_from_file_like(tmpdir, naturalearth_lowres, driver, ext):
         result2 = read(f)
 
     assert_equal_result((meta, index, geometry, field_data), result2)
+
+
+@pytest.mark.parametrize(
+    "driver,ext", [("GPKG", "gpkg"), ("FlatGeobuf", "fbg")]
```

Comment:
```suggestion
    "driver,ext", [("GPKG", "gpkg"), ("FlatGeobuf", "fgb")]
```

Reply:
Specifying driver should not be necessary; these extensions have good default behavior.